app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
```

Comment:
You can make something like `cmd, old_file, new_file = parts` after len() check to improve code readability